### PR TITLE
Fix vo_rpi fullscreen handling

### PR DESCRIPTION
**Pull request by cyph84**
Previous code was removed in 59cdfe50b2. This caused the OSD layer to be
not resized along with the video layer.

### CODE REVIEW

**Comment by sfan5** [comment]
Please read https://github.com/mpv-player/mpv/blob/master/DOCS/contribute.md, specifically the section about commit messages.

**Comment by cyph84** [comment]
Re-did commit message and updated. Thanks for pointing it out.

File: video/out/vo_rpi.c [hunk 1]
```diff
@@ -720,11 +721,28 @@ static struct mp_image *take_screenshot(struct vo *vo)
...
 
     switch (request) {
+    case VOCTRL_VO_OPTS_CHANGED: {
+        void *opt;
+        while (m_config_cache_get_next_changed(p->opts_cache, &opt)) {
+            struct mp_vo_opts *opts = p->opts_cache->opts;
+            if (&opts->fullscreen == opt)
+                set_fullscreen(vo);
+        }
+        return 1;
```
Comment (Dudemanguy) [inline video/out/vo_rpi.c:744]: Nit but for consistency's sake this should return `VO_TRUE`.

Comment (cyph84) [reply video/out/vo_rpi.c:744]: Noted. I based my code on b6fa63b01e, so should probably change that too.

Comment (Dudemanguy) [reply video/out/vo_rpi.c]: Hmm you're right, that VO mixes 1's, true's, and VO_TRUE (doesn't really matter but still).

**Review by Dudemanguy — APPROVED** [review]
LGTM. I assume this works as advertised.

**Comment by cyph84** [comment]
Just a note for anyone trying to patch out: you still need https://github.com/mpv-player/mpv/pull/9097 to get mpv to initialize properly with the MMAL libraries on the rpi.

**Pull request merged**